File: fix_wedding.php
<?php
$path = "d:\\Project\\temuruang\\resources\\views\\templates\\wedding\\wedding-02.blade.php";
$lines = file($path);

$new_lines = array_slice($lines, 0, 684);
$script_lines = array_slice($lines, 1012, 140);
$new_lines = array_merge($new_lines, $script_lines);
$new_lines[] = "</body>\n</html>\n";

file_put_contents($path, implode("", $new_lines));
echo "Fixed successfully!";
?>

File: fix_wedding.py
import sys

path = r"d:\Project\temuruang\resources\views\templates\wedding\wedding-02.blade.php"
with open(path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

new_lines = lines[:684]
new_lines.extend(lines[1012:1152])
new_lines.append("</body>\n</html>\n")

with open(path, 'w', encoding='utf-8') as f:
    f.writelines(new_lines)
print("Fixed successfully!")
